src/App.css

```diff
@@ -0,0 +1,59 @@
+body {
+    margin: 0;
+}
+
+.container {
+    min-height: 100vh;
+    max-height: 100%;
+    background-color: beige;
+    display: grid;
+    grid-template-columns: 150px 1fr;
+    grid-template-rows: 100px 100%;
```

Comment:
Using a percentage here is causing some white space at the bottom of the page. I think you're after `100px 1fr` instead?

Reply:
ah thank you! that works exactly how I need it do now